code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -216,16 +223,24 @@ var/const/MAX_ACTIVE_TIME = 400
 	if(ishuman(L))
 		var/mob/living/carbon/human/H = L
 		var/obj/item/mouth_protection = H.get_body_part_coverage(MOUTH)
+		if(!real && mouth_protection)
+			return //Toys really shouldn't be forcefully removing gear
+		var/obj/item/clothing/mask/facehugger/hugger = locate(/obj/item/clothing/mask/facehugger/ in contents)
```

Comment:
Wouldn't this allow you to carry a facehugger in your pockets or something to become facehugger-proof?